CHANGES.rst

```diff
@@ -22,6 +22,9 @@ migration of the IO plugins to `RosettaSciIO
 - :ref:`NeXus data format <nexus-format>`: The `dataset_keys`, `dataset_paths`
   and `metadata_keys` keywords were removed. Use `dataset_key`, `dataset_path`
   and `metadata_key` instead.
+- ``metadata.Signal.binned`` is removed, use the ``is_binned`` axis attribute
```

Comment:
This is the rsciio-split related API removal section.

When merging in Release next major, you can amend it to the other API removals.
(the duplicate `API Removal` headline has to be then removed as well)